GiEnJul/ClientApp/src/components/registerGiver/SignUp.tsx

```diff
@@ -1,179 +1,188 @@
-                    </div>
-                </Container>
-            )
-        case 3:
-            return (
-                <Container component="main" maxWidth="xs">
-                    <CssBaseline />
-                    <div className='paper'>
-                        <Typography component="h1" variant="h4">
-                            Bli giver
-                        </Typography>
-                        <Typography component="h2">
-                            Familiesammensetning
-                        </Typography>
-                        <LocationGiver
-                            nextStep={nextStep}
-                            prevStep={prevStep}
-                            handleLocationChange={handleFamilyChange}
-                            values={values}
-                            options={familiyOptions}
-                            placeHolder={'Familiestørrelse'}
-                        ></LocationGiver>
-                    </div>
-                </Container>
-            )
-        case 4:
-            return (
-                <Container component="main" maxWidth="xs">
-                    <CssBaseline />
-                    <div className='paper'>
-                        <Typography component="h1" variant="h4">
-                            Bli giver
-                        </Typography>
-                        <Typography component="h2">
-                            Oppsummering
-                        </Typography>
-                        <SummaryRegistration
-                            nextStep={nextStep}
-                            prevStep={prevStep}
-                            handleLocationChange={handleLocationChange}
-                            handlefullnameChange={handlefullnameChange}
-                            handleEmailChange={handleEmailChange}
-                            handleTlfChange={handleTlfChange}
-                            handleFamilyChange={handleFamilyChange}
-                            options={familiyOptions}
-                            locationOptions={locationOptions}
-                            submit={submit}
-                            values={values}
-                            callingback={handleConfirm}
-                        ></SummaryRegistration>
-                    </div>
-                </Container>
-            )
-        case 5:
-            return (
-                <Container component="main" maxWidth="xs">
-                    <CssBaseline />
-                    <div className='paper'>
-                        <Confirmation values={values} confirmationOK= {confirmationOK}></Confirmation>
-                    </div>
-                </Container>
-            )
-        default:
-            return(
-                <Container>
+  const handleConfirm = (submitOK: boolean) => {
+    setState((prev) => {
+      return { ...prev, confirmationOK: submitOK };
+    });
+  };
 
-                </Container>
-            )
-    };
```

Comment:
Theres a lot of unnecessary rewriting of Container tagm CSSBaseline tag and the div with classname paper. Make the switch to return something wrapped in a fragment (<></>) and then just set whatever is returned into the other three elements.